client/lib/plans/constants.js

```diff
@@ -860,6 +860,7 @@ export const FEATURES_LIST = {
 	[ FEATURE_ALL_FREE_FEATURES ]: {
 		getSlug: () => FEATURE_ALL_FREE_FEATURES,
 		getTitle: () => i18n.translate( 'All free features' ),
+		getDescription: () => i18n.translate( 'Also includes all features offered in the free plan.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Also includes all features offered in the Personal plan.' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/signup-plans-feature-info)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).